schemas/Organization.json

```
@@ -29,6 +29,48 @@
           "example": "https://mycallbackurlhandler.com/canopy"
         }
       }
+    },
+    "config": {
+      "description": "Configurations for integrations for your organization.",
+      "type": "object",
+      "properties": {
+        "issuer_processor": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "example": "PRIVACY",
+              "description": "The name of the active configured issuer-processor for your organization."
```

```suggestion
              "description": "The active issuer-processor for your organization."
```

should this be an enum? like payment processor "NONE" | "REPAY"? or is this only descriptive?

---

It's an enum in the Input --- I'm trying to minimize enums in the outputs because they're making it really tough for Charlie.